docs/supporting-artifacts/0020-LMS-Tables.sql

```diff
@@ -121,6 +121,11 @@ CREATE TABLE [lms].[LMSSection] (
     [SourceCreateDate] [DATETIME2](7) NULL,
     [SourceLastModifiedDate] [DATETIME2](7) NULL,
     [DeletedAt] [DATETIME2](7) NULL,
+    [EdFiLocalCourseCode] [NVARCHAR](60) NULL,
+    [EdFiSessionName] [NVARCHAR](60) NULL,
+    [EdFiSchoolYear] [SMALLINT] NULL,
+    [EdFiSchoolId] [INT] NULL,
+    [EdFiSectionIdentifier] [NVARCHAR](255) NULL,
```

Comment:
For efficiency, I was envisioning adding the unique identifier here instead of the entire natural key.

Reply:
Removed